manifests/agent/params.pp

```diff
@@ -0,0 +1,18 @@
+# Class: check_mk::agent::params
+#
+# @summary Sets the defaults for the agent class.
+# @api private
+#
+class check_mk::agent::params {
+    case $::osfamily {
+    'RedHat': {
+      $config_dir = '/etc/check-mk-agent'
+    }
+    'Debian': {
+      $config_dir = '/etc/check_mk'
+    }
+    default: {
+      fail("OS familly ${::osfamily} is not managed!")
```

Comment:
```suggestion
      fail("OS familly ${facts['os']['family']} is not managed!")
```

Reply:
OK.  I'll update `check_mk::params` as well as `check_mk::agent::params` (and fix the spelling of `family`!)

Reply:
Fixed